kt/entry-generation/godot-kotlin-symbol-processor/src/main/kotlin/godot/annotation/processor/utils/JvmTypeProvider.kt

```diff
@@ -0,0 +1,38 @@
+package godot.annotation.processor.utils
+
+import godot.entrygenerator.model.JvmType
+
+class JvmTypeProvider: (JvmType) -> Set<String> {
```

Comment:
Why not creating a top level lambda instead ?

Reply:
Just because of lazy initialization of top level members. Once the class file is loaded, the lambda stays around forever even if not used anymore.
In this case really not relevant as the whole lifecycle of it is bound to the build process anyways so i would be fine if we make a top level lambda out of it.